### ISSUE TITLE
[Accessibility] : Narrator / NVDA is reading as ‘Edit Read Only’ when navigated to Read-only TextField

### ISSUE THREAD

**Issue by v-sagsah** [microsoft/fluentui#12942]
**Browser Details:**
**OS:**  Windows 10 (Version 10.0.18363 Build 18363)
Microsoft Edge Beta Version 81.0.416.50 (Official build) beta (64-bit)
Google Chrome Version 81.0.4044.113 (Official Build) (64-bit)
**Screen Reader:** Narrator and NVDA

**Issue:**
Narrator / NVDA is reading as ‘Edit Read Only’ when navigated to Read-only TextField.

**Repro Steps :**

1. Start Narrator. Hit the URL "https://developer.microsoft.com/en-us/fluentui#/controls/web/textfield".
2. Tab till Read-only textfield control under usage section.
3. Verify what screen reader is reading.

**Below is a snapshot of the issue using NVDA + chrome**
![image](https://user-images.githubusercontent.com/64409491/80698953-d8150480-8af8-11ea-8a30-202770050aeb.png)

**Actual Result:**
Screen reader is reading as ‘Edit Read Only’ when navigated to Read-only TextField.

**Expected Result:**
Screen reader should not read as ‘Edit’ at ‘Read Only’ fields.

**User Impact:**
Screen reader users may find it difficult to understand the role of Read-only TextField when it read as ‘Edit’ at ‘Read Only’ fields.

**Comment by aneeshack4** [comment]
@v-sagsah I believe this is a screenreader problem because this Textfield has all the proper ARIA properties hooked up. I ran the [Accessibility Insights tool](https://accessibilityinsights.io/docs/en/web/overview) to double check and it didn't flag any errors/missing ARIA properties so it's followed the ARIA spec. So this is not actionable for us - I'm going to resolve as external.

**Comment by v-sagsah** [comment]
@aneeshack4  Thank You for the Info.
